Dockerfile

```diff
@@ -1,8 +1,9 @@
 FROM alpine:latest AS kafka_dist
```

Comment:
shouldn't we pin the alpine version? 

and defining an ARG with default so the CI build params can potentially build & validate multiple target versions.